Sources/SwiftGtk/enums/TreeViewColumnSizing.swift

```diff
@@ -0,0 +1,41 @@
+import CGtk
+
+/// The sizing method the column uses to determine its width. Please note that `GTK_TREE_VIEW_COLUMN_AUTOSIZE` are inefficient for large views, and can make columns appear choppy.
+/// 
+/// https://docs.gtk.org/gtk3/enum.TreeViewColumnSizing.html
+public enum TreeViewColumnSizing {
+    /// Columns only get bigger in reaction to changes in the model.
+    case growOnly
+
+    /// Columns resize to be the optimal size everytime the model changes.
+    case autosize
```

Comment:
Renaming this to `auto` might be a bit clearer (the `size` in `autosize` seems a bit redundant). In these cases I think it is ok to deviate from the Gtk naming, because it's still clear that `auto` is equivalent to Gtk's `autosize`.